.github/workflows/rust.yml

```diff
@@ -241,6 +241,7 @@ jobs:
 
       - name: Setup Python Script
         if: matrix.directory == 'packages/engine'
+        working-directory: ./packages/engine
         run: ${{ matrix.directory }}/src/worker/runner/python/setup.sh python3.7
```

Comment:
```suggestion
run: src/worker/runner/python/setup.sh python3.7
```

Reply:
I was expecting it to be an absolute path. No such luck

Reply:
`matrix.directory` is generated from `git --diff`, which is relative :)